contents/product-engineers/best-customer-retention-metrics.md

```diff
@@ -0,0 +1,246 @@
+
+To work out your net revenue churn, find your churned revenue in a period and subtract your expansion revenue and divide that total by your monthly recurring revenue in the previous period. Now multiply that new total by 100.
+
+For example, Hogflix had an MRR of $100,000 in May. In April, its churned revenue was $17,000 and its expansion revenue was $3,000. So subtract $3,000 from $17,000 to get $15,000 and divide that by $100,000 to get 0.15. Multiply that by 100 to get 15%.
+
+### What’s a good revenue churn rate?
+
+This depends again on how much your customers pay and how often they pay. If your business is based on monthly subscriptions, and your average revenue per account is $10 or less per month, then aim for monthly net revenue churn of around 6% or less. 
+
+The more customers pay, the lower your churn should be. At $250 average revenue per account per month, aim for a net revenue churn [of 1.4% or less](https://techcrunch.com/2022/04/20/study-up-on-churn-rate-basics-to-set-customer-and-revenue-benchmarks/), according to data compiled by analytics platform ChartMogul. This sort of churn rate also assumes that your customers are paying annually and, therefore, churn less often because they can’t ditch their contract.
+
+## 4. Net Promoter Score (NPS)
+
+### What is NPS? 
+
+Developed in the early-2000s and popularized by an article in _[Harvard Business Review](https://hbr.org/2003/12/the-one-number-you-need-to-grow)_, the Net Promoter Score continues to be a very popular way of measuring customer loyalty. It’s based on one question: ‘How likely are you to recommend [company/product/service] to a friend or colleague on a scale of 0 to 10?’Customers are then put into one of three buckets. 
+
+* **Promoters**, who gave a score of 9 or 10, are loyal enthusiasts and very likely to recommend you. 
+* **Passives**, who gave a score of 7 or 8, aren’t super happy or unhappy but are also unlikely to promote your business. 
+* **Detractors**, those who rated you between 0 and 6, could potentially damage your business through word-of-mouth.
+
+As you might imagine from a metric that relies on people answering a single question, the NPS is both alluring and [shaky](https://www.wsj.com/articles/the-dubious-management-fad-sweeping-corporate-america-11557932084). Alluring because, amongst a sea of often confusing data points it can seemingly provide clarity. But shaky because, well, a single data point isn’t enough to go on. As a result, it’s important to combine it with other metrics, including user surveys and interviews, to really understand what’s going on.
+
+### How is it useful?
+
+Pretty much all subscription-based businesses should be measuring their NPS as customer satisfaction is a key driver of churn. This is especially true for e-commerce businesses – if your online store is poorly designed and hard to use, your chances of repeat business will plummet. 
```

Comment:
```suggestion
Because it's an easy to understand, qualitative indicator of retention, and a simple way of measuring whether the changes you're making to your product to improve retention will be successful. If your NPS improve over time you can be reasonably confident retention should improve as well.
```